### DISCUSSION TITLE
Speech to Text (STT) Default Services

### DISCUSSION THREAD

**Discussion by will-rice** [home-assistant/architecture#516] [General] [answered]
I was redirected here from discord. 

I'm working on adding the [company](https://www.spokestack.io/) I work for as a speech to text platform and noticed that the internal integration doesn't have default services like the internal tts one does. Since I'm going to have to do a custom service, I was wondering if there was a plan for default speech-to-text services. If there is a plan, I could just implement the default ones instead of doing a custom one, so everyone could benefit from it.

**Comment by MartinHjelmare** [comment, marked as answer]
There's no plan for default services at the moment.
